Formula/m/meilisearch.rb

```diff
@@ -5,9 +5,12 @@ class Meilisearch < Formula
   sha256 "1c13850d1735aaaad6e3ff0281ae8ea5868c6a08de04c2c3ee163cadf1dcb434"
   license "MIT"
 
+  # Upstream creates releases that use a stable tag (e.g., `v1.2.3`) but are
+  # labeled as "pre-release" on GitHub before the version is released, so it's
+  # necessary to use the `GithubLatest` strategy.
```

Comment:
```suggestion
  # There can be a notable gap between when a version is tagged and a
  # corresponding release is created, so we check the "latest" release instead
  # of the Git tags.
```

There isn't a 1.6.0 release yet, so livecheck is currently returning 1.6.0 as newest simply because of the `v1.6.0` tag. From what I'm seeing, upstream only labels unstable versions as "pre-release" (`v1.6.0-rc.0`, `v1.5.0-rc.3`, etc.), so this is just another instance where there's a gap between when the tag and release are created.

That said, `GithubLatest` is the right approach and we just need to update the explanatory comment.